Makefile

```diff
@@ -20,9 +20,10 @@ UP_BIN=$(BIN_DIR)/up
 OTELCOL_BIN=$(BIN_DIR)/otelcol
 MEMCACHED_BIN=$(BIN_DIR)/memcached
 PROMETHEUS_BIN=$(BIN_DIR)/prometheus
+PROMTOOL_BIN=$(BIN_DIR)/promtool
 GOJSONTOYAML_BIN=$(BIN_DIR)/gojsontoyaml
 JSONNET_BIN?=$(BIN_DIR)/jsonnet
-JSONNETFMT_BIN?=$(BIN_DIR)/jsonnetfmt
+SONNETFMT_BIN?=$(BIN_DIR)/jsonnetfmt
```

Comment:
```suggestion
JSONNETFMT_BIN?=$(BIN_DIR)/jsonnetfmt
```

Reply:
Oops. Accidental edit, will amend this into the current commit to limit noise in commit history.